alert-policies/coredns/coredns-panic-rate.yml

```diff
@@ -0,0 +1,69 @@
+# Name of the alert
+name: CoreDNS Panics
+
+# Description and details
+description: |+
+  This alert will trigger if the CoreDNS panic rate is above 0 at least once in 5 minutes
+
+# Type of alert
+type: BASELINE
+
+# NRQL query
+nrql:
+  # Baseline alerts can use an optional FACET
+  query: "FROM Metric SELECT rate(sum(coredns_panics_total), 1 SECONDS) as 'Panic rate' WHERE instrumentation.name = 'remote-write' and prometheus_server is not null facet instance, prometheus_server"
+
+# Direction in which baseline is set (Default: LOWER_ONLY)
+baselineDirection: STATIC
```

Comment:
this isn't a valid baseline direction; the API only accepts `LOWER_ONLY`, `UPPER_ONLY`, and `UPPER_AND_LOWER` - should this be a `type: STATIC` alert?